File: python-sdk/src/sdk/models/shared/generalevent.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class GeneralEvent:
    asdfzxcv: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('asdfzxcv') }})
    distinct_id: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('distinct_id') }})
    event: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('event') }})
    

File: ts-sdk/src/sdk/models/shared/generalevent.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class GeneralEvent extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=asdfzxcv" })
  asdfzxcv?: string;

  @SpeakeasyMetadata({ data: "json, name=distinct_id" })
  distinctId?: string;

  @SpeakeasyMetadata({ data: "json, name=event" })
  event?: string;
}
